components/stepper/steps/overview.md

```diff
@@ -86,26 +86,52 @@ To include the desired steps for the Stepper component, you can either manually
 }
 ````
 
-## Features
+## Parameters
```

Comment:
Maybe it makes more sense to describe the parameters first, and then provide a demo. Test this approach and see if you like it more.